packages/auth/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "@ctrlplane/db": "workspace:*",
-    "@auth/drizzle-adapter": "^1.1.0",
+    "@auth/drizzle-adapter": "1.4.1",
```

Comment:
This needed to be hard coded to fix this issue: https://github.com/nextauthjs/next-auth/issues/11490